---
title: "Metabarcoding model development in `NIMBLE`"
author: "A. J. Rominger"
date: "`r format(Sys.time(), '%d %B %Y')`"
output: pdf_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE, comment='##')
```

# The model

We know the total number of reads $N_{reads}$, the number of species $S$, the total number of individuals $N$, and the correlation structure from the phylogeny $\Sigma_{phy}$.  We also know the vector of number of reads per species $x_{read}$. We do not know the rate of copy number evolution $\sigma_{copy}$ nor the rate of primer affinity evolution $\sigma_{primer}$.  We also most importantly don't know the actual vector of abundances $x$.  We want to model the number of reads as multinomial conditioned on $N_{reads}$ and $S$, and with a probability vector $p$ given by
$$
p \sim \text{dir}(x \nu)
$$
where $\nu$ is a variable proportional to copy number.  We will assume that $\text{log}\nu$ evolved according to a Brownian motion process with rate $\sigma_{copy}$ and mean $\mu_0 = 0$.

The variance-covariance matrix for a Brownian-motion process on a known phylogeny is given as
$$
\Sigma_{i,j} = \sigma D_{MRCA(i,j)}
$$
where $D_{MRCA(i,j)}$ is the distance from the root to the most recent common ancestor of tips $i$ and $j$.  $D_{MRCA(i,i)}$ is simply the depth of the root.

Thus our full model is 
$$
x_{reads} \sim \text{multinom}(N_{reads}, p_1, \ldots, p_S)
$$
where 
$$
p \sim \text{dir}(x \nu)
$$
and $\nu$ is distributed
$$
\text{log}(\nu) \sim \text{mvnorm}(0, \sigma_{copy} d_{MRCA(i,j)})
$$

The priors for this hierrarchical model are
$$
x/N \sim \text{dir}(N/S, \ldots, N/S)
$$
$$
\frac{1}{\sigma^2_{copy}} \sim \Gamma(0.001, 0.001)
$$


# `NIMBLE` model

We first simulate some data that will be needed in the model specification:

* `S` number of species to simulate
* `tre` is the underlying phylogeny
* `N` is the number of actual individuals
* `Nreads` is the total number of reads
* `numberReads` is a vector of the number of reads assigned to each spp

```{r, message=FALSE, cache=FALSE}
# simulation functions
source('~/Dropbox/hawaiiDimensions/mol2ecol/simMetaBar.R')

# set parameters for simulation
S <- 70
set.seed(1)
n <- sample(round(seq(1, 80, length.out = S)))
N <- sum(n)
Nreads <- 1e+06
sigCopy <- 0.1

# simulate phylogeny and number of reads resulting form metabarcoding
set.seed(1)
sim <- simMetaBar(abund = n, sigCopy = sigCopy, nreads = Nreads)

# extract needed objects from output
numberReads <- sim$reads
tre <- sim$tre
ordID <- sim$ordID
```

Let's quickly check the simulation to see that it's reasonable

```{r, fig.height=3, fig.width=6, cache=FALSE}
palette(hsv(h = seq(0, 0.8, length.out = max(ordID)), 
            s = 1-0.3*seq(-1, 1, length.out = max(ordID))^2, 
            v = 0.7 + 0.3*seq(-1, 1, length.out = max(ordID))^2))
par(mfrow = c(1, 2), mar = c(3, 3, 0, 0) + 0.5, mgp = c(2, 0.75, 0))
plot(sim$abund, numberReads, xlab = 'Real abundance', ylab = 'Simulated number of reads', 
     col = ordID, cex = 1.5, pch = 16)
plot(tre, show.tip.label = FALSE)
axis(1, at = pretty(par('usr')[1:2]), labels = rev(pretty(par('usr')[1:2])))
mtext('Mya', side = 1, line = 2)
```

To specify our model in `NIMBLE` we can define and use custom multinomial-Dirichlet functions:

```{r, results='hide', message=FALSE}
library(nimble)

ddirchmulti <- nimbleFunction(
    run = function(x = double(1), alpha = double(1), size = double(0),
                   log = integer(0, default = 0)) {
        returnType(double(0))
        alpha0 <- sum(alpha)
        
        # new log prob that ignores 0's instead of throwing NaN/Inf
        lgammaSum <- numeric(length = length(x), value = 0, init = TRUE)
        for(i in 1:length(lgammaSum)) {
            if(x[i] > 0) {
                lgammaSum[i] <- log(x[i]) + lgamma(alpha[i]) +
                    lgamma(x[i]) -
                    lgamma(alpha[i] + x[i])
            }
        }
        logProb <- log(size) +
            lgamma(alpha0) + lgamma(size) - lgamma(alpha0 + size) -
            sum(lgammaSum)
        
        if(log) return(logProb)
        else return(exp(logProb))
    }
)

rdirchmulti <- nimbleFunction(
    run = function(n = double(0, default = 1), alpha = double(1), size = double(0)) {
        returnType(double(1))
        
        # modified from MCMCpack to allow alpha_k = 0
        x <- numeric(length = length(alpha), value = 0, init = TRUE)
        for(i in 1:length(x)) x[i] <- rgamma(1, shape = alpha[i], rate = 1)
        p <- x/sum(x)
        
        return(rmulti(1, size = size, prob = p))
    }
)
```

The arguments are:
    
* `x` vector of values (e.g. number of reads for each species $x_{reads}$)
* `alpha` vector of parameters of the Dirichlet distribution
* `size` number of trials (e.g. total number of reads $N_{reads}$)
* `n` number of observations (only `n = 1` supported)


We can now specify the model:

```{r, cache=FALSE}
mod <- nimbleCode({
    # p is proportion of total abundance for each spp
    # x[1:S] <- N*p[1:S]
    
    # define phylogenetically-correlated copy number (nu) and primer affinity (lambda)
    # Note: tauCopy is inverse variance, and 'Prec' stands for 'Preceission' matrix (the
    # inverse of the var-covar matrix)
    # Note: var-cov matrix D is defined in constants, as is mu0 (which is a vector of 0)
    PrecCopy[1:nOrd, 1:nOrd] <- tauCopy * Dinv[1:nOrd, 1:nOrd]
    log(nu[1:nOrd]) ~ dmnorm(mu0[1:nOrd], PrecCopy[1:nOrd, 1:nOrd])
    
    # define dirichlet-multinom params and the distribution of x_{reads}
    for(i in 1:S) {
        alpha[i] <- x[i] * nu[ordID[i]]
    }
    xreads[1:S] ~ ddirchmulti(alpha[1:S], Nreads)
    
    # priors
    x[1:S] ~ dmulti(p0[1:S], N) # p0 defined in constants
    tauCopy ~ dgamma(0.001, 0.001)
})
```


And initialize the model

```{r, cache=FALSE}
# model constants, data and inits
modConstants <- list(S = S, N = N, Nreads = Nreads, nOrd = max(ordID), ordID = ordID,
                     Dinv = solve(vcv(tre)),  # vcv gives the vcov mat of a phylo for sig = 1
                     mu0 = rep(0, max(ordID)), p0 = rep(1/S, S))

modData <- list(xreads = numberReads)

modInits <- list(
    # priors
    tauCopy = 1,
    x = {
        foo <- rep(round(N / S), S)
        foo[0:(N - sum(foo))] <- foo[0:(N - sum(foo))] + 1
        foo
    }, 
    
    # deterministic relationships
    PrecCopy = modConstants$Dinv,
    
    # hyper distribution
    nu = rep(1, modConstants$nOrd)
)

# build model
mod <- nimbleModel(code = mod, name = 'mod', constants = modConstants, 
                   data = modData, inits = modInits)
```

Now we can compile and run the model

```{r, chache=TRUE}
Cmod <- compileNimble(mod)
modConf <- configureMCMC(mod)
modConf$addMonitors('alpha')
modConf$addMonitors('nu')

# set the thinning param
thin <- 1
modConf$setThin(thin)

# add block sampling for x
modConf$addSampler(target = sprintf('nu[1:%s]', modConstants$S),
                   type = 'RW_block')

# compile MCMC
modMCMC <- buildMCMC(modConf)
CmodMCMC <- compileNimble(modMCMC, project = mod)

# set MCMC params
mcmcN <- 1e+03
burn <- 1
niter <- (mcmcN + burn) * modConf$thin
CmodMCMC$run(niter)

# the posterior sample    
samp <- as.matrix(CmodMCMC$mvSamples)[-(1:burn), ]
```

All sampled parameters are saved in `samp`.  We can evaluate stationarity of our MCMC with some diagnostics.  First just trace plots:

```{r, fig.width=6, fig.height=3, cache=FALSE}
par(mfrow = c(1, 2), mar = c(3, 3, 0, 0) + 0.5, mgp = c(2, 0.75, 0))
plot(1/samp[, 'tauCopy'], type = 'l', xlab = 'iter', ylab = 'sigCopy')
abline(h = sigCopy, col = 'red')
plot(samp[, 'x[3]'], type = 'l', xlab = 'iter', ylab = 'Sp 2 abund')
abline(h = sim$abund[2], col = 'red')
```

Now auto-correlation plots
```{r, fig.width=6, fig.height=3, cache=FALSE}
par(mfrow = c(1, 2), mar = c(3, 3, 0, 0) + 0.5, mgp = c(2, 0.75, 0))
acf(1/samp[, 'tauCopy'])
acf(samp[, 'x[2]'])
```

Now a plot of real versus estimated abundance
```{r, fig.width=3, fig.height=3, cache=FALSE}
par(mar = c(3, 3, 0, 0) + 0.5, mgp = c(2, 0.75, 0))
plot(sim$abund, colMeans(samp[, grep('x', colnames(samp))]), 
     xlab = 'real abundance', ylab = 'estimated abundance')
abline(0, 1, col = 'red')
```
